test/integration/api/policies_controller_test.rb

```diff
@@ -7,7 +7,7 @@ def setup
     @provider = FactoryBot.create(:provider_account)
     @service = @provider.default_service
     login! @provider
-    Policies::PoliciesListService.expects(:call!)
+    Policies::PoliciesListService.expects(:call!).returns({}.to_json)
```

Comment:
No! I was wrong and deleted the comment. Just `{}` is fine.

```suggestion
    Policies::PoliciesListService.expects(:call!).returns({})
```